apps/web/src/components/registration-table.tsx

```diff
@@ -0,0 +1,294 @@
+          )) ||
+      (groupFilter === "programmerbar" &&
+        registration.user.memberships.some(
+          (membership) => membership.group?.name === "Programmerbar",
+        )) ||
+      (groupFilter === "hovedstyret" &&
+        registration.user.memberships.some(
+          (membership) => membership.group?.name === "Hovedstyret",
+        )) ||
+      (groupFilter === "makerspace" &&
+        registration.user.memberships.some(
+          (membership) => membership.group?.name === "Makerspace",
+        )) ||
+      (groupFilter === "tilde" &&
+        registration.user.memberships.some(
+          (membership) => membership.group?.name === "Tilde",
+          )) ||
+      (groupFilter === "esc" &&
+        registration.user.memberships.some(
+          (membership) => membership.group?.name === "Esc"));
+
+    return matchesSearchTerm && matchesYearFilter && matchesStatusFilter && matchesGroupFilter;
+  });
+
+  const resetFilters = () => {
+    setSearchTerm("");
+    setYearFilter("");
+    setStatusFilter("");
+    setGroupFilter("");
+  };
+
+  return (
+    <div className="relative overflow-x-auto border shadow-md sm:rounded-lg">
+
+      <div className="relative overflow-x-auto pt-2">
+        <div className="flex w-full gap-5 p-4 pt-0">
+          <div className="flex flex-col">
+            <Label className="px-2">Søk:</Label>
+            <input
+              className="sm:rounded-lg"
+              type="text"
+              placeholder="Søk etter navn eller e-post"
+              value={searchTerm}
+              onChange={(e) => setSearchTerm(e.target.value)}
+            />
+          </div>
+          <div className="flex flex-col">
+            <Label className="px-2">Trinn:</Label>
+            <select
+              className="sm:rounded-lg"
+              value={yearFilter}
+              onChange={(e) => setYearFilter(e.target.value)}
+            >
+              <option value="">Alle</option>
+              <option value="1">1</option>
+              <option value="2">2</option>
+              <option value="3">3</option>
+              <option value="4">4</option>
+              <option value="5">5</option>
+            </select>
+          </div>
+          <div className="flex flex-col">
+            <Label className="px-2">Status:</Label>
+            <select
+              className="sm:rounded-lg"
+              value={statusFilter}
+              onChange={(e) => setStatusFilter(e.target.value)}
+            >
+              <option value="">Alle</option>
+              <option value="påmeldt">Bare påmeldt</option>
+              <option value="venteliste">Bare venteliste</option>
+              <option value="avmeldt">Bare avmeldt</option>
+              <option value="fjernet">Bare fjernet</option>
+            </select>
+          </div>
+
+          <div className="flex flex-col">
+            <Label className="px-2">Undergruppe:</Label>
+            <select
```

Comment:
Bruk `<Select />`. https://github.com/echo-webkom/new-echo-web-monorepo/pull/860.

For å hente alle undergruppene, kunne du ha brukt databasen til å de ut? Så er de ikke hardkodet.

Reply:
@NixEngh kan du fikse?